update-cache/README.md

```diff
@@ -0,0 +1,25 @@
+#AMP-Toolbox update-cache
+
+Generates update-cache invalidation URLs, as described on the AMP documentation at 
+https://developers.google.com/amp/cache/update-ping#update-cache-request.
```

Comment:
* can you turn this into a proper link 
* the link doesn't point to the AMP documentation

Reply:
Couldn't find the update-cache spec o the AMP doc. I think it's specific to the Google AMP Cache. 

The AMP Cache guidelines state:

"8. Supports a public Update Cache mechanism for document publishers to notify the AMP cache about new, updated or deleted documents:

Equivalent to the Google AMP Update Cache API"

https://github.com/ampproject/amphtml/blob/00679960b7fd8e68f2ff159f84fe01bc8ea00a0f/spec/amp-cache-guidelines.md

Reply:
Changed to "Google AMP Cache documentation" instead of "AMP documentation"